orchestration/dagster_orchestration/hca_orchestration/pipelines/load_hca.py

```diff
@@ -72,17 +83,96 @@
         "load_tag": load_tag,
         "scratch_config": scratch_config,
         "target_hca_dataset": target_hca_dataset,
-        "bigquery_service": mock_bigquery_service
+        "bigquery_service": mock_bigquery_service,
+        "slack": console_slack_client,
+        "dagit_config": preconfigure_resource_for_mode(dagit_config, "test")
+    }
+)
+
+
+def _base_slack_blocks(title: str, key_values: dict[str, str]) -> list[dict[str, object]]:
+    return [
+        {
+            "type": "section",
+            "text": {
+                'type': 'mrkdwn',
+                'text': f'*{title}*',
+            }
+        },
+        {
+            'type': 'divider'
+        },
+        {
+            'type': 'section',
+            'fields': [
+                {
+                    'type': 'mrkdwn',
+                    'text': '\n'.join(map(lambda keys: f'*{keys}*', key_values.keys())),
+                },
+                {
+                    'type': 'mrkdwn',
+                    'text': '\n'.join(key_values.values())
+                }
+            ]
+        }
+    ]
+
+
+@success_hook(
+    required_resource_keys={'slack', 'target_hca_dataset', 'dagit_config'}
+)
+def import_start_notification(context: HookContext) -> None:
+    context.log.info(f"Solid output = {context.solid_output_values}")
+    job_id = context.solid_output_values["result"]
+    kvs = {
+        "Staging Area": context.solids.pre_process_metadata.input_prefix,
+        "Target Dataset": context.resources.target_hca_dataset.dataset_name,
+        "Jade Project": context.resources.target_hca_dataset.project_id,
+        "TDR Job ID": job_id,
+        "Dagit link": f'<{context.resources.dagit_config.run_url(context.run_id)}|View in Dagit>'
     }
+
+    context.resources.slack.send_message(blocks=_base_slack_blocks("HCA Starting Import", key_values=kvs))
+
+
+@failure_hook(
+    required_resource_keys={'slack', 'target_hca_dataset', 'dagit_config'}
+)
+def import_failed_notification(context: HookContext) -> None:
+    if "result" in context.solid_output_values:
```

Comment:
🔧 Same comment about job_id in this context--we won't have it. Remove the references to job_id here as well.